doc/source/io.rst

```diff
@@ -2604,6 +2604,26 @@ table CSS classes. Note that these classes are *appended* to the existing
 
    print(df.to_html(classes=['awesome_table_class', 'even_more_awesome_class']))
 
+The ``render_links`` argument provides the ability to add hyperlinks to cells
+that contain URLs.
+
+.. ipython:: python
+
+   url_df = pd.DataFrame({
+       'name': ['Python', 'Pandas'],
+       'url': ['https://www.python.org/', 'http://pandas.pydata.org']})
+   print(url_df.to_html(render_links=True))
+
+.. ipython:: python
+   :suppress:
+
+   write_html(url_df, 'render_links', render_links=True)
```

Comment:
what is this for?

Reply:
This runs the write_html() call without including its input or output in the notebook cells that appear in the doc. This makes available the static file renderd on line 2625.

I copied a pattern used elsewhere in this file, example "bold_rows" section, starting line 2584.

Reply:
What's the downside of just printing it? I think creating files like this is hard to keep track of within our codebase so less than ideal

Reply:
The generated HTML is also printed in a notebook cell; the lines pointed out by @jreback write that HTML to a separate file that can be rendered.

Here's what it looks like after building:

![image](https://user-images.githubusercontent.com/8574029/49829597-b1a7cb00-fd43-11e8-96ff-4a54bb7f1ea1.png)

Reply:
hmm i am not sure we actually need to show the html, showing the rendered version is enough